app/Http/Controllers/Front/PostController.php

```diff
@@ -0,0 +1,249 @@
+<?php
+
+namespace App\Http\Controllers\Front;
+
+use App\Http\Controllers\Front\PostbaseController;
+use Illuminate\Http\Request;
+use Intervention\Image\Facades\Image;
+use Illuminate\Support\Facades\File;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Support\Facades\Input;
+use Illuminate\Support\Facades\Hash;
+use App\User;
+use App\Post;
+
+class PostController extends PostbaseController {
+
+
+    public function index() {
+
+        $data = [];
+        $data['menu_post'] = true;
+        return view('posts', $data);
+    }
+
+    public function load(Request $request) {
+        // if ajax request
+        if ($request->ajax()) {
+            $user_id = $this->globaldata['user']->user_id;
```

Comment:
`$request->user()->id` returns the same thing here and is coming from the injected `$request` rather than a super global - which makes it easier to read.